code/game/objects/items/devices/multitool.dm

```diff
@@ -158,14 +158,16 @@
 		M.toggle_hud(owner)
 	return 1
 
-/obj/item/multitool/cyborg
-	name = "multitool"
-	desc = "Optimised and stripped-down version of a regular multitool."
-	toolspeed = 0.5
-
 /obj/item/multitool/abductor
 	name = "alien multitool"
 	desc = "An omni-technological interface."
 	icon = 'icons/obj/abductor.dmi'
 	icon_state = "multitool"
 	toolspeed = 0.1
+
+/obj/item/multitool/cyborg
+	name = "electronic multitool"
+	desc = "Optimised version of a regular multitool. Streamlines processes handled by its internal microchip."
+	icon = 'icons/obj/items_cyborg.dmi'
+	icon_state = "multitool_cyborg"
+	toolspeed = 0.5

```

Comment:
revert the move, the only diff should be 171/172

Reply:
why? i'll do it but from my pov i'm only cleaning the code up a bit and making it easier to read by standarizing a bit where the tools are in each .dm

Reply:
solved with latest commit